src/nuke-sw.js

```diff
@@ -0,0 +1,16 @@
+function run() {
+  // Is this actually being executed in a ServiceWorker?
+  if (!self.clients && !self.registration) {
+    return;
+  }
+  // Nuke the service worker.
+  self.registration.unregister();
+
+  // Reload all open pages.
+  self.clients.matchAll().then(clients => {
```

Comment:
```suggestion
  clients.matchAll({ includeUncontrolled: true }).then(clients => {
```

This service worker isn't controlling pages, so you need the above to match all pages. I guess this could be turned into an `await` too.